docs/developer-guide/ksqldb-reference/scalar-functions.md

```diff
@@ -40,6 +40,26 @@ CREATE TABLE AGG AS
    GROUP BY ID;
 ```
 
+### CAST
+
+Since: -
+
+```sql
+CAST(COL0 AS BIGINT)
+```
+
+Converts one type to another. The following casts are supported:
+
+| from | to | notes |
+|------|----|-------|
+| ANY  | STRING | Converts the type to its string representation. |
+| STRING | BOOLEAN | Any string that exactly matches `true`, case-insensitive, will be converted to `true`. Any other value will be converted to `false`. |
+| STRING | INT, BIGINT, DECIMAL, DOUBLE | Converts string representation of numbers to number types. Conversion will fail if text does not contain a number or the number does not fit in the indicated type. |
```

Comment:
```suggestion
| STRING | INT, BIGINT, DECIMAL, DOUBLE | Converts string representation of numbers to number types. Conversion fails if text doesn't contain a number or the number doesn't fit in the indicated type. |
```

Reply:
Does "fail" mean throw an exception?

Reply:
Sure, but 'throws an exception' doesn't necessarily mean anything to the user of ksqlDB.  This isn't a library where the dev using it needs to know to expect an exception.